scripts/qa_crowbarsetup.sh

```diff
@@ -622,6 +622,8 @@ function onadmin_prepare_cloud_repos()
             develcloud5)
                 addsp3testupdates
                 add_sles12ga_testupdates
+                addcloud5maintupdates
```

Comment:
The official build numbers are calculated differently, so its not uncommon for them to be higher than develcloud5.